public/pyscript/main.py

```
@@ -1,101 +1,114 @@
+    modified_image.save(stream, format=actual_fmt.upper())
+    stream_val = stream.getvalue()
+    js_array = Uint8Array.new(len(stream_val))
+    js_array.assign(stream_val)
+
+    # Create a JS File object. Name and type are important for blob URL and download.
+    return File.new([to_js(js_array)], name, to_js({"type": f"image/{actual_fmt.lower()}"}))
+
+# display_image_file function is removed as results are dispatched via interop
+
+@when("process-images", ".image-bleed-processor")
+async def process_files(event):
```

Ruff (0.11.9)

Missing return type annotation for public function `process_files`

Add return type annotation: `None`

(ANN201)